pageserver/src/consumption_metrics.rs

```diff
@@ -5,13 +5,14 @@
 //!
 
 use anyhow;
+use pageserver_api::models::TenantState;
 use tracing::*;
 use utils::id::NodeId;
 use utils::id::TimelineId;
 
+use crate::context::RequestContext;
 use crate::task_mgr;
 use crate::tenant::mgr;
-use pageserver_api::models::TenantState;
```

Comment:
Was going to comment about moving `pageserver_api` imports, but... seems like we're pretty inconsistent anyways about whether we group `pageserver_api` with imports from the same crate or external crates.

Reply:
Yeah, cleaned it up.